manifests/plugin/write_graphite/carbon.pp

```diff
@@ -12,6 +12,7 @@
   $protocol                  = 'tcp',
   Boolean $separateinstances = false,
   Boolean $logsenderrors     = true,
+  $reconnectinterval         = 0,
```

Comment:
Can you please add a suitable datatype for this parameter?

Reply:
Done, thanks